### PR TITLE
docs(readme): link to @excalidraw/excalidraw npm package to give more visibility

### PR DESCRIPTION
**Pull request by ad1992**
Adding link to the main readme to increase the visibility of the package since it resides in the same repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ If you like the project, you can become a sponsor at [Open Collective](https://o
 
 <a href="https://opencollective.com/excalidraw#category-CONTRIBUTE" target="_blank"><img src="https://opencollective.com/excalidraw/tiers/backers.svg?avatarHeight=32"/></a>
 
+## Embed Excalidraw in your App ?
```
Comment (lipis) [inline README.md:31]:
```suggestion
## Embed Excalidraw in your App?
```

File: README.md [hunk 2]
```diff
@@ -28,6 +28,10 @@ If you like the project, you can become a sponsor at [Open Collective](https://o
...
+
+Try out [@excalidraw/excalidraw](https://www.npmjs.com/package/@excalidraw/excalidraw).
```
Comment (lipis) [inline README.md:33]:
```suggestion
Try out [`@excalidraw/excalidraw`](https://www.npmjs.com/package/@excalidraw/excalidraw).
```

Comment (lipis) [reply README.md:33]: But probably we need more text here.

**Comment by lipis** [comment]
What about adding it before `Development`?

File: README.md [hunk 3]
```diff
@@ -70,6 +70,10 @@ The second set of digits is the encryption key. The Excalidraw server doesn’t
 
 Find a growing list of libraries containing assets for your drawings at [libraries.excalidraw.com](https://libraries.excalidraw.com).
 
+## Embed Excalidraw in your App?
```
Comment (dwelle) [inline README.md:73]:
```suggestion
## Building an app using Excalidraw?
```

If not the above, let's change `Embed` → `Embedding`?

Comment (ad1992) [inline README.md:73]:
```suggestion
## Embedding Excalidraw in your App?
```

Comment (ad1992) [reply README.md]: The latter sounds better :)

**Review by lipis — APPROVED** [review]

**Comment by lipis** [comment]
We could create a sample project using it..

**Comment by ad1992** [comment]
> We could create a sample project using it.

We already have [code sandbox example](https://codesandbox.io/s/excalidraw-v3-wcqdl) no, we can extend that if needed? or you are talking about hosting it ourselves completely?

**Comment by lipis** [comment]
oh... didn't know.. I was thinking about a separate repo or something.. `excalidraw-app`, `excalidraw-example-app`, `excalirdraw-starting-pack-using-npm`..

**Comment by ad1992** [comment]
> oh... didn't know..

That gives a signal we might need to improve the readme :D 

>  I was thinking about a separate repo or something.. `excalidraw-app`, `excalidraw-example-app`, `excalirdraw-starting-pack-using-npm`..

yep, I was thinking of having an example in our repo itself so it will also help in testing/ developement as well.

**Comment by lipis** [comment]
> yep, I was thinking of having an example in our repo itself so it will also help in testing/ developement as well.

We could have example for every feature that it offers.. it makes sense to have a dead simple example.

**Review by lipis — APPROVED** [review]

**Pull request merged**